.cirrus.yml

```diff
@@ -101,13 +159,50 @@ task:
   environment:
     CLOUDSMITH_API_KEY: ENCRYPTED[!2cb1e71c189cabf043ac3a9030b3c7708f9c4c983c86d07372ae58ad246a07c54e40810d038d31c3cf3ed8888350caca!]
 
+  libs_cache:
+    folder: build/libs
+    fingerprint_script: echo "`md5 lib/CMakeLists.txt` macos"
+    populate_script: make libs build_flags=-j8
+
   install_script:
     - brew install coreutils python
     - pip3 install --upgrade cloudsmith-cli
 
   release_script:
     - bash .ci-scripts/x86-64-apple-darwin-release.bash ${CLOUDSMITH_API_KEY}
 
+task:
+  only_if: $CIRRUS_TAG =~ '^\d+\.\d+\.\d+$'
+
+  windows_container:
+    dockerfile: .ci-dockerfiles/windowsservercore/Dockerfile
+    os_version: 2019
+    cpu: 8
+    memory: 24
+
+  name: "nightly: x86-64-pc-windows-msvc"
```

Comment:
this name is incorrect. for a tag, that isn't a nightly. it should be `release:` not `nightly:`

Reply:
Oops, will fix.